Site/bc_weather_station_dashboard/website/templates/weather.html

```diff
@@ -131,9 +143,10 @@ <h2 class="card-text" id="wind-speed">N/A</h2>
                     </div>
                     <!-- Wind Direction Card -->
                     <div class="col-md-4">
-                        <div class="card rounded shadow mb-md-0 mb-4">
-                            <div class="card-body text-center" id="wind-direction">
```

Comment:
removing this messed up the wind direction widget

Reply:
I am seeing the same problem

Reply:
I readded the lines of code needed for wind direction widget, I don't even know how those got deleted... Good catch!